data/scenarios/02Tutorials/02-craft.yaml

```diff
@@ -1,29 +1,33 @@
 name: Crafting, part 1
+description: |
+  Learn how to make new things using recipes.
 entities:
   - name: goal
     display:
       attr: device
       char: 'X'
     description:
-    - |
-      Robots can use the 'make' command to make things.
-      Your base has a tree, check the inventory for the
-      available recipes. Your goal is to create a "logger".
+    - Robots can use the 'make' command to make things.
+    - Your base has a few trees in its inventory, select them to see the
+      available recipes.
+    - Your goal is to make a "branch predictor", so you will have to make
+      some "branches" first.
 win: |
   try {
-    as base {has "logger"}
+    as base {has "branch predictor"}
   } { return false }
 solution: |
-  make "log"; make "logger"
+  make "branch"; make "branch predictor"
```

Comment:
Hey @TristanCacqueray, sorry to change your tutorial, but I realised that previously the player would not see errors.

So I changed it to "branch predictor" and I made the description a bit more explicit.

Also, I gave players a few more trees to play with, but they will have to get all of them in the next challenge :smiling_imp: